wandb/sdk/wandb_init.py

```diff
@@ -144,6 +144,8 @@ def setup(self, kwargs):
             self._jupyter_setup(settings)
 
         self._log_setup(settings)
+        if settings.root_dir is None:
+            settings.root_dir = os.path.abspath(os.getcwd())
```

Comment:
This seems odd that everything even worked before...  How did we set root_dir when it wasn't specified before?